test/unit/driver/test_linode.py

```diff
@@ -0,0 +1,194 @@
+        '-o StrictHostKeyChecking=no'
+    ]
+
+    assert options == _instance.default_ssh_connection_options
+
+
+def test_login_options(_instance, mocker):
+    m = mocker.patch('molecule.driver.linode.Linode._get_instance_config')
+
+    m.return_value = {
+        'instance': 'linode',
+        'address': '172.16.0.2',
+        'user': 'linode-admin',
+        'port': 22,
+        'identity_file': '/foo/bar',
+    }
+
+    x = {
+        'instance': 'linode',
+        'address': '172.16.0.2',
+        'user': 'linode-admin',
+        'port': 22,
+        'identity_file': '/foo/bar',
+    }
+
+    assert x == _instance.login_options('linode')
+
+
+def test_ansible_connection_options(_instance, mocker):
+    m = mocker.patch('molecule.driver.linode.Linode._get_instance_config')
+
+    m.return_value = {
+        'instance': 'linode',
+        'address': '172.16.0.2',
+        'user': 'linode-admin',
+        'port': 22,
+        'ssh_pass': 'foobar',
+        'identity_file': '/foo/bar',
+    }
+
+    x = {
+        'ansible_host':
+        '172.16.0.2',
+        'ansible_port':
+        22,
+        'ansible_user':
+        'linode-admin',
+        'ansible_private_key_file':
+        '/foo/bar',
+        'connection':
+        'ssh',
+        'ansible_ssh_common_args': ('-o UserKnownHostsFile=/dev/null '
+                                    '-o ControlMaster=auto '
+                                    '-o ControlPersist=60s '
+                                    '-o IdentitiesOnly=yes '
+                                    '-o StrictHostKeyChecking=no'),
+        'ansible_ssh_pass':
+        'foobar',
+    }
+
+    assert x == _instance.ansible_connection_options('linode')
+
+
+def test_instance_config_property(_instance):
+    x = os.path.join(_instance._config.scenario.ephemeral_directory,
+                     'instance_config.yml')
+
+    assert x == _instance.instance_config
+
+
+def test_ssh_connection_options_property(_instance):
+    options = [
+        '-o UserKnownHostsFile=/dev/null', '-o ControlMaster=auto',
+        '-o ControlPersist=60s', '-o IdentitiesOnly=yes',
+        '-o StrictHostKeyChecking=no'
+    ]
+
+    assert options == _instance.ssh_connection_options
+
+
+def test_status(mocker, _instance):
+    result = _instance.status()
+
+    assert 2 == len(result)
+
+    assert result[0].instance_name == 'instance-1'
+    assert result[0].driver_name == 'linode'
+    assert result[0].provisioner_name == 'ansible'
+    assert result[0].scenario_name == 'default'
+    assert result[0].created == 'false'
+    assert result[0].converged == 'false'
+
+    assert result[1].instance_name == 'instance-2'
+    assert result[1].driver_name == 'linode'
+    assert result[1].provisioner_name == 'ansible'
+    assert result[1].scenario_name == 'default'
+    assert result[1].created == 'false'
+    assert result[1].converged == 'false'
+
+
+def test_created(_instance):
+    assert 'false' == _instance._created()
```

Comment:
why is it a string?

Reply:
Unsure but [after looking at this](https://github.com/ansible/molecule/blob/master/molecule/driver/base.py#L202-L206), I don't think I'll be changing anything from the base class for this PR.